bigchaindb/voting.py

```diff
@@ -0,0 +1,152 @@
+import collections
+
+from bigchaindb.common.schema import SchemaValidationError, validate_vote_schema
+from bigchaindb.common.utils import serialize
+from bigchaindb.common.crypto import PublicKey
+
+
+VALID = 'valid'
+INVALID = 'invalid'
+UNDECIDED = 'undecided'
+
+
+class Voting:
```

Comment:
I started out with just functions, but then I used classmethods so it's easy to create a derived implementation that overrides one function (to modify the schema for example).